docs/release-notes/2.28.0.md

```diff
@@ -0,0 +1,107 @@
+
+## New Features
+
+- [PMM-9749](https://jira.percona.com/browse/PMM-9849): Advisors: Possibility to run individual advisor checks separately
+
+- [PMM-9469](https://jira.percona.com/browse/PMM-9469): Advisors: Ability to have multiple queries in a single check
+
+- [PMM-9468](https://jira.percona.com/browse/PMM-9468): Advisors: Ability to query VictoriaMetrics as a data source.
+
+## Improvements
+
+- [PMM-9841](https://jira.percona.com/browse/PMM-9841): Advisors: Advisor checks are now enabled by default.
+
+- [PMM-8326](https://jira.percona.com/browse/PMM-8326): Advisors: Changed the icon for the **Edit Check Rule** option with a more suggestive one that better reflects this functionality.
+
+- [PMM-9907](https://jira.percona.com/browse/PMM-9907): pmm2-client now supports Ubuntu 22.04 LTS.
+
+- [PMM-9780](https://jira.percona.com/browse/PMM-9780): VictoriaMetrics has been upgraded to 1.76.1.
+
+- [PMM-5871](https://jira.percona.com/browse/PMM-5871): Node Exporter has now been updated to 1.3.1.
+
+- [PMM-9958](https://jira.percona.com/browse/PMM-9958): The PMM logs button, which is used to download PMM logs for troubleshooting, is added to the help panel for better accessibility and enhanced user experience.
+
+## Bugs Fixed
+
+- [PMM-9854](https://jira.percona.com/browse/PMM-9854): Advisors: In some scenarios, PMM was not displaying the complete list of advisors available for instances connected to Percona Platform. This issue is now fixed.
+
+- [PMM-9848](https://jira.percona.com/browse/PMM-9848): Advisors: Fixed text contrast issue on the **Failed Advisor Checks** page that was visible when navigating the list of results while using PMM with the Light theme.
+
+- [PMM-9426](https://jira.percona.com/browse/PMM-9426): DBaaS: Fixed an issue related to K8s monitoring where the k8s monitoring failed with k8s version 1.22 and higher.
+
+- [PMM-9885](https://jira.percona.com/browse/PMM-9885): Dashboard: Fixed the documentation links on the **Advanced settings** page on the PMM dashboard.
+
+- [PMM-9828](https://jira.percona.com/browse/PMM-9828): Fixed an issue with the QAN dashboard navigator/explorer where if you open QAN from a dashboard and try to navigate to a different dashboard, the explorer keeps closing/refreshing, making it impossible to navigate.
+- [PMM-9363](https://jira.percona.com/browse/PMM-9363): PMM users logged in via SSO would still have access to PMM after disconnecting. This issue is now fixed and PMM correctly terminates SSO sessions after disconnecting.
+
+- [PMM-9415](https://jira.percona.com/browse/PMM-9415): Fixed an issue where initial data restore on AWS instances fails. However, consecutive data restore attempts were successful.
+
+
+## Known Issues
+
+**Issue**
+
+`origin not allowed` error is thrown while accessing the PMM dashboard after upgrading to PMM 2.27.0 and onwards when using a reverse proxy like Nginx.
```

Comment:
@nikita-b Please check if this is correct

Reply:
@rasika-chivate let's add it for 2.27.0 release notes too.

And I think we can rephrase it:

If you are using a reverse proxy (for example nginx) in front of PMM you can see an error: `origin not allowed` after updating to PMM 2.27.0 or newer.